frontend/src/components/annotation/audio_annotation/render_elements/waveform_selector.vue

```diff
@@ -132,17 +144,19 @@ export default Vue.extend({
 
         if ( instance ) {
           const { start_time, end_time, label_file } = instance
-          if (!this.invisible_labels.includes(label_file.id)) {
+          if (!this.invisible_labels.includes(label_file.id) && !instance.soft_delete ) {
```

Comment:
Delete command does not remove the instance completely, but sets `soft_delete` property on it, so need to use it here to control rendering of annotation regions.